packages/react-native/ReactCommon/react/renderer/components/image/ImageEventEmitter.cpp

```diff
@@ -13,24 +13,49 @@ void ImageEventEmitter::onLoadStart() const {
   dispatchEvent("loadStart");
 }
 
-void ImageEventEmitter::onLoad() const {
-  dispatchEvent("load");
+void ImageEventEmitter::onLoad(const ImageSource& source) const {
+  dispatchEvent("load", [=](jsi::Runtime& runtime) {
```

Comment:
```suggestion
  dispatchEvent("load", [source](jsi::Runtime& runtime) {
```

nit: prefer explicit capture group so it is clear what is being copied into closure

Reply:
Er, I see existing code is also doing this, so eh